scripts/build.sh

```diff
@@ -1,2 +1 @@
-node_modules/.bin/babel lib/ --out-dir dist
-node_modules/.bin/coffee -co dist/ lib/
+babel lib/ --out-dir dist/ --presets es2015,stage-0,react
```

Comment:
:+1: yeah, it's confusing to spread config across multiple places so whenever it's possible to consolidate, let's do that.